script.js

```diff
@@ -0,0 +1,44 @@
+let books = JSON.parse(localStorage.getItem('books')) || [];
+function addBook(title, author) {
+  const book = { title, author };
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+function removeBook(title) {
+  books = books.filter((book) => book.title !== title);
+  localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- [ ] [REQUIRED] Good job so far! When I try to `remove` one of the `added options`, the remaining added options remain removed. Please correct this 🙏🏼 

![mobile (1)](https://github.com/mumbereh/AwesomeBook_pair_project/assets/63915024/6e4f45f0-6ef1-404b-9575-8de28085b228)